homeassistant/components/jellyfin/const.py

```diff
@@ -2,6 +2,8 @@
 
 from typing import Final
 
+from ...const import Platform
```

Comment:
To keep this in line with the rest of the code
```suggestion
from homeassistant.const import Platform
```